br/br-pitr-manual.md

```diff
@@ -392,6 +392,6 @@ Restore KV Files <--------------------------------------------------------------
 
 > **Note:**
 >
-> - When you restore the cluster for the first time, you must specify the full snapshot data. Otherwise, some data in the newly created table might be incorrect due to rewriting Table ID rules.
+> - When you restore the cluster for the first time, you must specify the full snapshot data. Otherwise, some data in the newly created table might be incorrect due to rewriting Table ID rules. For more information, see issue [#54418](https://github.com/pingcap/tidb/issues/54418).
```

Comment:
```suggestion
> - When you restore the cluster for the first time, you must specify the full snapshot data. Otherwise, some data in the newly created table might be incorrect due to rewriting Table ID rules. For more information, see GitHub issue [#54418](https://github.com/pingcap/tidb/issues/54418).
```